docs/dictionary-and-concepts/dictionary-metering.md

```diff
@@ -12,43 +12,61 @@ The terms listed here are all terms relating to the topics of defining metering.
 | Exchange Point |  A domain for establishing energy exchange between two Metering Grid Areas. **Additional information:** This is a type of Metering Point.|
 | **F** |   |
 | **G** |   |
 | **H** |   |
 | **I** |   |
+| Industry Classification | Indicates the industry context, e.g. E23 = Electricity |
 | In Area | Indicates which grid an exchange point transports energy into   |
 | **J** |   |
 | **K** |   |
 | **L** |   |
 | **M** |   |
 | Meter ID | ID for a physical device containing one or more registers.|
-| Metering Grid Area |  A Metering Grid Area is a physical area where consumption, production and exchange can be measured. It is delimited by the placement of meters for continuous measurement for input to, and withdrawal from the area. **Additional information:** It can be used to establish volumes that cannot be measured such as network losses. |
-| Metering Point | An entity where energy products are measured or computed.|
 | Meter Reading Periodicity| The frequency for which a metered data will be submitted |
 | Meter Type |  |
+| Metering Grid Area |  A Metering Grid Area is a physical area where consumption, production and exchange can be measured. It is delimited by the placement of meters for continuous measurement for input to, and withdrawal from the area. **Additional information:** It can be used to establish volumes that cannot be measured such as network losses. |
+| Metering Point | An entity where energy products are measured or computed |
+| Metering Point ID | A unique metering point identifier |
+| Metering Point Type | Indicates the type of a metering point, e.g. Exchange |
 | MP Description | An option that can be used to describe how a physical meter can be located, e.g. basement |
 | Multiplier |  Meter conversion |
 | **N** |   |
 | **O** |   |
-| Out Area |  Indicates which grid an exchange point transports energy out of   |
+| Observation Time | A calculated value that indicates a date and time for a position in a time series period |
+| Out Area |  Indicates which grid an exchange point transports energy out of |
 | **P** |   |
 | Parent Metering Point | A term used when a Metering Point, or Accounting Point becomes part of a Metering Point Parent-Child hierachy, where the Parent acts as the top-level Metering Point. |
+| Point | One point equals one measurement within a time series. If a metering point is hourly read, a time series for a single day contains 24 points (Except for days switching between daylight saving time and standard time) |
+| Position | Indicates the position of a measurement within a time series period |
+| Product | An energy product identification, e.g. active energy |
 | **Q** |   |
-| Quantity | Quantity of a specific type of energy  |
+| Quantity | Quantity of a specific type of energy |
+| Quality | Indicates the quality of a specific quantity in a time series, e.g. estimated, measured, etc. |
```

Comment:
Should it be possible to see all possible values somewhere in the documentation?

Reply:
I would like to have that somewhere. I think we need a common approach to this across teams.